docs/release-notes/NuGet-5.0-preview.md

```diff
@@ -9,11 +9,57 @@ ms.topic: conceptual
 
 # NuGet 5.0 Preview Release Notes
 
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version| Available in .NET SDK(s)|
+|:---|:---|:---|
+| [**5.0.0-p4**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.0.0-preview4/rc](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview) | [2.1.600, 2.2.200](https://www.microsoft.com/net/download/visual-studio-sdks) |
+
```

Comment:
For the .NET SDK, those versions are not available on the page you point to.  I updated the VS link to point to the preview release notes that also has the download link for VS preview.